### PR TITLE
Added Haptic Feedback: UX Enhancement

### PR DESCRIPTION
**Pull request by gokulnair2001**
# Reason
Hi Rudrank Riyam , I saw your App on App Store, played it and found it really interesting app. The only drawback which I saw was the app is not that good in terms of User Experience. So this made me to think about that slight feature which your app was lagging behind and that was "Haptic Feedback ". 

So I added the Haptic Feedback feature in the app, therefore from now when the user clicks any option he will ge the haptic Feedback from the phone.

# Attention 
Hope so you liked my enhancement towards Unofficial-SF-Symbols-Game. Kindly accept my request because I really feel this feature would help your app to enhance the UX.

# Changes Done:
One new file added: HapticFeedback.swift 
One new one of code in the mainSymbolView.swift file.

### Feedback
Kindly give me a reply, no matter be positive or negative news :).

### CODE REVIEW

File: Unofficial SF Symbols Game/Unofficial SF Symbols Game/Utilities/HapticFeedback.swift [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+
+class haptickFeedback {
+    
+    // Declaring class as static
+    static let feedback = haptickFeedback()
+
+    // Haptic Feedback method
+    func haptiFeedback() {
+  let generator = UINotificationFeedbackGenerator()
+  generator.notificationOccurred(.success)
```
Comment (rudrankriyam) [inline Unofficial SF Symbols Game/Unofficial SF Symbols Game/Utilities/HapticFeedback.swift:20]: Ctrl + I for re-indenting the lines. :)

File: Unofficial SF Symbols Game/Unofficial SF Symbols Game/Views/Main View/MainSymbolGameView.swift [hunk 2]
```diff
@@ -39,15 +42,15 @@ class MainSymbolGameViewModel: ObservableObject {
 
 struct MainSymbolGameView: View {
     @ObservedObject var viewModel: MainSymbolGameViewModel
-
+    
     var body: some View {
         VStack {
             Text("SF Symbols Game").largeTitleText(topPadding: 50)
-
+
```
Comment (rudrankriyam) [inline Unofficial SF Symbols Game/Unofficial SF Symbols Game/Views/Main View/MainSymbolGameView.swift:49]: Looks like unwanted spacing here

**Review by rudrankriyam — CHANGES_REQUESTED** [review]

File: Unofficial SF Symbols Game/Unofficial SF Symbols Game/Views/Main View/MainSymbolGameView.swift (hunk 2 above)
Comment (gokulnair2001) [reply Unofficial SF Symbols Game/Unofficial SF Symbols Game/Views/Main View/MainSymbolGameView.swift:49]: sorry will do it and push the file

**Comment by rudrankriyam** [comment]
Hi @gokulnair2001! I'm glad that you liked the game! And yes, the app is in a 0.1 release state. I'll give it some love when I can find time in the next few weeks.

And haptics does improve the experience! Thank you for your contribution! I've added a review for the small nitpicky thing, otherwise looks good to me!

**Comment by gokulnair2001** [comment]
Thanks a lot for the reply, will surely make the changes and push the file

**Review by rudrankriyam — APPROVED** [review]

**Comment by gokulnair2001** [comment]
Thank you so much for accepting the PR :) .

**Pull request merged**